### PR TITLE
Build for arm64

### PR DESCRIPTION
**Pull request by mrueg**
This adds builds for arm64 as well.

### CODE REVIEW

File: .github/workflows/docker-build.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Docker build/push
       uses: docker/build-push-action@v2
       with:
-        platforms: linux/amd64,linux/arm/v7
+        platforms: linux/amd64,linux/arm/v7, linux/arm64
```
Comment (sfudeus) [inline .github/workflows/docker-build.yaml:36]:
```suggestion
        platforms: linux/amd64,linux/arm/v7,linux/arm64
```

Comment (sfudeus) [reply .github/workflows/docker-build.yaml:36]: Could you additionally add the new architecture into the README.md? At the end, there is a list of supported architectures.

Comment (mrueg) [reply .github/workflows/docker-build.yaml]: Thanks, done!

**Comment by sfudeus** [comment]
Perfect, thx.

**Pull request merged**